Task 2/Task 2.1/Task 2.1.2/Figures/Polygons/Triangle.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Task_2._1._2.Figures.Polygons
+{
+    public class Triangle : Polygon
+    {
+        public Triangle(Point p1, Point p2, Point p3) : base("Треугольник", p1, p2, p3) { }
+
+        public override double Area()
```

Comment:
So, all these figures are immutable (that is okay) and their public interface only allows to know their name, perimeter and area?(that is not okay)
Imagine you use these figures in a real graphics editor - how editors code will know where to draw these figures?